### ISSUE TITLE
User set volume should be saved in client local storage

### ISSUE THREAD

**Issue by robartsd** [ArnoldSmith86/virtualtabletop#1016]
This was brought up on discord due to the volume resting when the browser window is reloaded for any reason - server reset, conflicting delta, user refresh - none of these should cause a change to the local volume setting.

**Comment by robartsd** [comment]
Should this be saved on a per-room or per-server basis?

**Comment by bjalder26** [comment]
I agree that this would be an improvement.  The current default volume is somewhat arbitrary (it used to be based on the idea that you could double the volume from that point if the volume was set to 0.5 in the JSON, but I wound up moving away from that idea).  So, I would lean towards a per-server basis, because the default is pretty arbitrary anyway.  Either that, or maybe revisit the default too (just a number change).
